react/features/base/participants/functions.js

```diff
@@ -122,6 +122,36 @@ export function getParticipantCount(stateful: Object | Function) {
     return getParticipants(stateful).length;
 }
 
+/**
+ * Returns participant's display name.
+ * FIXME: remove the hardcoded strings once interfaceConfig is stored in redux.
+ *
+ * @param {(Function|Object)} stateful - The (whole) redux state, or redux's
+ * {@code getState} function to be used to retrieve the state.
+ * @param {string} id - The ID of the participant's display name to retrieve.
+ * @private
+ * @returns {string}
+ */
+export function getParticipantDisplayName(
```

Comment:
There's similar method used on web, but it we won't be able to use it on RN, because it generates HTML content. Can we mention it in the FIXME that it's to be merged eventually ? https://github.com/jitsi/jitsi-meet/blob/master/conference.js#L1119

Reply:
👍

Reply:
I don't see that one fixed or am I missing something ?

Reply:
Doh, my bad, I missed that! Will fix in a jiffy.

Reply:
Done.